### PR TITLE
changed requirements for upgrade to django==1.8, modified Person model i...

### PR DESCRIPTION
**Pull request by AmaxJ**
...n students app.

Let me know if this looks right.

### CODE REVIEW

File: rmotr_sis/students/models.py [hunk 1]
```diff
@@ -18,8 +18,8 @@ class Meta:
 class Person(TimeStampedModel):
 
     email = models.EmailField(max_length=150, unique=True)
-    first_name = models.CharField(max_length=150, blank=True, null=True)
-    last_name = models.CharField(max_length=150, blank=True, null=True)
+    first_name = models.CharField(max_length=150, blank=False, null=True)
```
Comment (martinzugnoni) [inline rmotr_sis/students/models.py]: I think there was a reason for all fields not being required. Probably related with legacy data imported from google spreadsheet were fields were not normalized.
^^ @santiagobasulto

Comment (santiagobasulto) [inline rmotr_sis/students/models.py]: https://imgflip.com/i/kgie3

Make everything nullable/blankable for now.

**Comment by santiagobasulto** [comment]
Good PR @AmaxJ. How did it go with migrations? They're a pain in the ass.

**Comment by AmaxJ** [comment]
The migrations went fine actually. Wont be a problem to revert the models back.

**Comment by AmaxJ** [comment]
By the way, sorry I mixed in different kinds of changes into this pull request. It should have just been the version upgrade alone.

**Comment by martinzugnoni** [comment]
@AmaxJ after removing the nullable changes you forgot to delete the new migration. It's not needed any more.

**Comment by martinzugnoni** [comment]
Gogogo :rocket:

**Pull request merged**